app/services/dataAccess/TestDataAccess.java

```diff
@@ -0,0 +1,44 @@
+package services.dataAccess;
+
+import services.dataAccess.proto.PostListProto.PostList;
+import services.dataAccess.proto.PostProto.Post;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Created by maria on 04/11/16.
+ */
+
+public class TestDataAccess extends AbstractDataAccess {
```

Comment:
Is making actual sample post objects in here to send out part of a different pull request?